public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -29,6 +37,29 @@
     private AchillesResultDao achillesResultDao;
     @Autowired
     private DbDomainDao dbDomainDao;
+    @Autowired
+    private ConceptService conceptService;
+
+    // TODO: consider putting this in CDM config, fetching it from there
+    private static final ImmutableMultimap<Domain, String> DOMAIN_MAP =
+            ImmutableMultimap.<Domain, String>builder()
+                    .put(Domain.CONDITION, "Condition")
+                    .put(Domain.CONDITION, "Condition/Meas")
```

Comment:
One thing of note: at least in our synthetic data, I've found these Condition/Meas, etc. domains to have deprecated / not useful concepts. I'm not sure if this will be true in the real data. Perhaps we can avoid using them entirely.